lib/l10n/intl_en.arb

```diff
@@ -28,6 +28,21 @@
   "@app_errorScreen_unknown_showStackTrace": {
     "type": "text"
   },
+  "app_logOut_confirm": "Log out",
+  "@app_logOut_confirm": {
+    "type": "text",
+    "description": ""
+  },
+  "app_logOut_content": "You'll need your login credentials to log in again.",
```

Comment:
Don't use contractions in a formal context. Maybe change this to "You will need to provide your login credentials to use the app again."

Reply:
Will do